index.js

```diff
@@ -60,6 +62,15 @@ async function generateModels(aspect, containerSelector, model, env=`live`) {
 
     // eslint-disable-next-line no-undef
     const wowModelViewer =  await new WowModelViewer(models)
+
+    // override wowhead events listeners
+    if ( disableFullScreen === true ){
```

Comment:
`=== true` should not be necessary, same for below

Reply:
I've just edit the way of doing it, let's check new code. Thank for review :)